geospatial/tests/test_events.py

```diff
@@ -0,0 +1,106 @@
+from http import HTTPStatus
+
+import pytest
+import datetime
+
+from geospatial.content.models import HomePage
+from geospatial.content.models import AuthorIndexPage
+from geospatial.content.models import AuthorPage
+from geospatial.content.models import EventIndexPage
+from geospatial.content.models import EventPage
+
+pytestmark = [pytest.mark.django_db]
+
+
+def test_event_index_page(client):
+    homepage = HomePage.objects.get()
+    event_index = EventIndexPage(
+        title='Events',
+        slug='events',
+    )
+    homepage.add_child(instance=event_index)
+    event = EventPage(
+        title='My Fancy Event',
+        slug='my-fancy-event',
+        intro='a short abstract',
+        body='a whole lot of text here',
+        date=datetime.date.today() + datetime.timedelta(days=1)
+    )
+    event_index.add_child(instance=event)
+
+    assert event_index.url == '/events/'
+    response = client.get(event_index.url)
+    assert response.status_code == HTTPStatus.OK
+    html = response.content.decode('utf8')
+    assert event.title in html
+    assert f'href="{event.url}"' in html
+    assert event.intro in html
+    assert event.body not in html
+
+
+def test_event_page(client):
+    homepage = HomePage.objects.get()
+    event_index = EventIndexPage(
+        title='Events',
+        slug='events',
+    )
+    homepage.add_child(instance=event_index)
+    event = EventPage(
+        title='My Fancy Event',
+        slug='my-fancy-event',
+        intro='a short abstract',
+        body='a whole lot of text here',
+        date=datetime.date.today() + datetime.timedelta(days=1)
+    )
+    event_index.add_child(instance=event)
+
+    assert event.url == '/events/my-fancy-event/'
+    response = client.get(event.url)
+    html = response.content.decode('utf8')
+    assert event.title in html
+    assert event.intro in html
+    assert event.body in html
+    assert event.date in html
```

Comment:
This is probably going to fail when you introduce Romanian date formatting. But you can check for the literal string value that you expect.